charts/castai-pod-node-lifecycle/values.yaml

```diff
@@ -0,0 +1,148 @@
+# Values used to connect to the CAST AI API.
+api:
+  url: https://api.cast.ai
+  key:
+  clusterId:
+
+replicaCount: 3
+
+image:
+  repository: castai/pod-node-lifecycle
```

Comment:
All our components should point to google artifact registry. If we don't have it in registry yet this can be done in separate task.

Reply:
Thanks!
We don't have it there yet, I've created a task for that.